resources/public/api/conf/1.0/retrieve_annual_submission.yaml

```diff
@@ -80,6 +80,8 @@ responses:
             $ref: './common/errors.yaml#/components/examples/ruleTaxYearNotSupported'
           RULE_TAX_YEAR_RANGE_INVALID:
             $ref: './common/errors.yaml#/components/examples/ruleTaxYearRangeInvalid'
+          RULE_INCORRECT_GOV_TEST_SCENARIO:
```

Comment:
Are we supposed to be adding this to the v1.0 endpoints too? As you've updated the documentation here but not the endpoint controller mapping (+ tests) for v1.0 - only 2.0